test/routes/github/index.spec.js

```diff
@@ -458,6 +458,36 @@ describe('server', () => {
 					`*${forkOwner}* just forked 🍴 *${repositoryFullName}*`,
 				);
 			});
+
+			it('sends a message to Twitch Chat', async () => {
+				const twitchChatSpy = jest.spyOn(TwitchChat.prototype, 'send');
```

Comment:
You may need to spy `StreamLabs` too, otherwise when you add the missing await is going to fail ;P